### ISSUE TITLE
Collect prisma scenes for Bali

### ISSUE THREAD

**Issue by echeipesh** [azavea/nasa-hyperspectral#90]
- [x] collect the HDF5 files for Bali
- [ ] provide summary for processing levels and what it means for us

**Comment by jpolchlo** [comment]
Documentation on PRISMA levels:

L0a
- the raw data, encoded in custom binary format
- 16-bit digital number (DN) at-sensor radiance
- contains cloud cover percentage estimate
- Data delivered in a variety of custom files

L1
- Radiometrically-calibrated radiance data
- frames are assembled into data cubes—one each for VNIR, SWIR, and PAN
- Coregistered data are created by interpolation
  - Non-coregistered data are included which are not interpolated—L0a data with radiance calibration
- generates cloud coverage, sun glint and generic land cover masks
- File stored as monolithic HDF-EOS5 file

Level 2 has several processing levels.  Common features are
- atmosphere-corrected, bottom-of-atmosphere
- metadata are adjusted for geolocation

L2b
- Radiance from bottom of atmosphere

L2c
- Reflectance ratio from bottom of atmosphere
- Aerosol characterization
- Column water vapor amounts over clear land pixels and above clouds
- Cloud optical thickness

L2d
- Geocoded version of L2c product
- Orthorectification uses surface geometry

In short, L0a is way too low level, so ignore this.  L1 gives TOA radiance.  L2 gives BOA radiance/reflectance.  L2c contains the greatest amount of information about the atmosphere.  L2d is the most geographically accurate.

**Comment by jpolchlo** [comment]
Closing this, @echeipesh, now that the levels are documented.  Reopen if I missed something.
